tests/lib/prepare-utils.sh

```diff
@@ -158,7 +158,15 @@ download_core24_snaps() {
 
     # download neccessary images
     # TODO: publish 24 kernel
-    snap download pc-kernel --channel=24/${snap_branch} --basename=upstream-pc-kernel
+    case "${snap_branch}" in
```

Comment:
this warrants a comment

Reply:
Added a comment. Removed the TODO. Added a fixme.